service/evaluation/evaluation.go

```diff
@@ -592,7 +696,7 @@ func (svc *Service) evaluateSubcontrol(toe *orchestrator.TargetOfEvaluation, con
 			// We let the scheduler running if we do not get the assessment results from the orchestrator, maybe it is
 			// only a temporary network problem
 			log.Errorf("could not get assessment results for Cloud Service ID '%s' and MetricIds '%s' from Orchestrator: %v", toe.GetCloudServiceId(), getMetricIds(metrics), err)
-		} else if len(assessmentResults) == 0 {
+		} else if len(assessments) == 0 {
 			// We let the scheduler running if we do not get the assessment results from the orchestrator, maybe it is
 			// only a temporary network problem
```

Comment:
I think in this case no assessment results are available an no network errors are present.